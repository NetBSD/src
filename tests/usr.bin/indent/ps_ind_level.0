/* $NetBSD: ps_ind_level.0,v 1.2 2021/10/05 17:12:20 rillig Exp $ */
/* $FreeBSD$ */
/* $ */
/* $ The leading indentation in the first line determines the indentation */
/* $ of the remaining code. In this particular test, the indentation is set */
/* $ to 5 and the tabulator is set to 8, to demonstrate an off-by-one error */
/* $ in main_prepare_parsing that has been fixed in indent.c 1.107 from */
/* $ 2021-10-05. */
/* $ */
/* $ The declaration in the below line is indented by 3 tabs, amounting to */
/* $ 24 spaces. The initial indentation of the code is intended to be */
/* $ rounded down, to 4 levels of indentation, amounting to 20 spaces. */
			int declaration_in_column_25;

/* $ Even if later code has a smaller indentation, it is nevertheless */
/* $ indented to the level given by the first line of code. */
void function_in_column_1(void){}
