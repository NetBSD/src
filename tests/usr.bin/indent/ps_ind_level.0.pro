/* $NetBSD: ps_ind_level.0.pro,v 1.1 2021/10/05 17:09:45 rillig Exp $ */
/* $FreeBSD$ */

-i5		/* 5 spaces indentation */
-ts8		/* 8 spaces per tabulator */
