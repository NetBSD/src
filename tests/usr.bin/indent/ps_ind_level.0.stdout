/* $NetBSD: ps_ind_level.0.stdout,v 1.1 2021/10/05 17:09:45 rillig Exp $ */
/* $FreeBSD$ */
/* $ */
/* $ FIXME: indentation should be 20 instead of 25. */
			 int		 declaration_in_column_25;

/* $ FIXME: The space between '){' is missing. */
			 void		 function_in_column_1(void){
			 }
